src/sql/parts/modelComponents/toolbarLayout.css

```diff
@@ -62,4 +62,12 @@
 	margin-right: 0.3em;
 	background-position: 50% 50%;
 
-}
\ No newline at end of file
+}
```

Comment:
What is this animation and why do we need it?

Reply:
It matches the behavior of the built-in SQL toolbar. Any extension will now behave the same as our standard toolbar so it's consistent.